modules/manually-maintained-credentials-upgrade.adoc

```diff
@@ -61,6 +80,27 @@ It may take several minutes after adding the annotation for the upgradeable stat
 
 . To view the CCO status details, click *cloud-credential* in the *Cluster Operators* list.
 
-. If the *Upgradeable* status in the *Conditions* section is *False*, verify that the `upgradeable-to` annotation is free of typographical errors.
-
+.. If the *Upgradeable* status in the *Conditions* section is *False*, verify that the `upgradeable-to` annotation is free of typographical errors.
+ifndef::ibm-cloud[]
 When the *Upgradeable* status in the *Conditions* section is *True*, you can begin the {product-title} upgrade.
+endif::ibm-cloud[]
+ifdef::ibm-cloud[]
++
+When the *Upgradeable* status in the *Conditions* section is *True*, you can begin the {product-title} upgrade.
+
+. Revoke the old service IDs and API Keys:
++
+[source,terminal]
++
```

Comment:
You don't need the line break between [source,terminal] and the start of the  code block

Reply:
Done.